app/services/resource_service.rb

```diff
@@ -142,6 +142,7 @@ def update_resource
 
     def unchanged_attributes?
       return false if @old_resource.blank?
+
```

Comment:
?

Reply:
habe es extra in einen eigegen commit gepackt:

style: fehlende Leerzeile hinzugefügt